org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/ColorBox.java

```diff
@@ -0,0 +1,32 @@
+/******************************************************************************
+ * Copyright (C) 2008 Low Heng Sin                                            *
```

Comment:
not sure where you copy this from but it should be remove

Reply:
Duplicated from FilenameBox